zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -133,14 +147,27 @@ where
             address_book,
             peer_service,
             next_peer_min_wait: sleep(Duration::from_secs(0)),
+            next_update_time: Instant::now(),
         }
     }
 
     /// Update the peer set from the network, using the default fanout limit.
     ///
     /// See [`update_initial`][Self::update_initial] for details.
+    ///
+    /// ## Security
+    ///
+    /// This call is rate-limited to prevent sending a burst of repeated requests for new peer
+    /// addresses to the same peers. Each call will only update the [`CandidateSet`] if more time
+    /// than [`MIN_PEER_GET_ADDR_INTERVAL`][Self::MIN_PEER_GET_ADDR_INTERVAL] has passed since the
+    /// last call.
```

Comment:
1. I don't think we need to say "the same peers" here, for similar reasons to my previous comment
2. Let's explicitly say that the update is skipped (rather than delayed using a sleep)
```suggestion
    /// This call is rate-limited to prevent sending a burst of repeated requests for new peer
    /// addresses. Each call will only update the [`CandidateSet`] if more time
    /// than [`MIN_PEER_GET_ADDR_INTERVAL`][Self::MIN_PEER_GET_ADDR_INTERVAL] has passed since the
    /// last call. Otherwise, the update is skipped.
```